operation/fhir-operation-cqf/src/main/java/com/ibm/fhir/operation/cqf/MeasureSubmitDataOperation.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.operation.cqf;
+
+import static com.ibm.fhir.cql.helpers.ModelHelper.*;
+
+import java.util.List;
+
+import com.ibm.fhir.cql.helpers.ParameterMap;
+import com.ibm.fhir.exception.FHIROperationException;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.MeasureReport;
+import com.ibm.fhir.model.resource.OperationDefinition;
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.resource.Resource;
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.model.type.code.BundleType;
+import com.ibm.fhir.model.type.code.HTTPVerb;
+import com.ibm.fhir.registry.FHIRRegistry;
+import com.ibm.fhir.server.operation.spi.AbstractOperation;
+import com.ibm.fhir.server.operation.spi.FHIROperationContext;
+import com.ibm.fhir.server.operation.spi.FHIRResourceHelpers;
+
+public class MeasureSubmitDataOperation extends AbstractOperation {
+
+    public static final String PARAM_IN_MEASURE_REPORT = "measureReport";
+    public static final String PARAM_IN_RESOURCE = "resource";
+    public static final String PARAM_OUT_RETURN ="return";
+    
+    @Override
+    protected OperationDefinition buildOperationDefinition() {
+        return FHIRRegistry.getInstance().getResource("http://hl7.org/fhir/OperationDefinition/Measure-submit-data", OperationDefinition.class);
+    }
+
+    @Override
+    public Parameters doInvoke(FHIROperationContext operationContext, Class<? extends Resource> resourceType, String logicalId, String versionId,
+        Parameters parameters, FHIRResourceHelpers resourceHelper) throws FHIROperationException {
+
+        ParameterMap paramMap = new ParameterMap(parameters);
+        
+        Parameter param = paramMap.getSingletonParameter(PARAM_IN_MEASURE_REPORT);
+        MeasureReport measureReport = (MeasureReport) param.getResource();
+        
+        List<Parameter> resources = paramMap.getParameter(PARAM_IN_RESOURCE);
+
+        Bundle.Builder builder = Bundle.builder()
+                .type(BundleType.TRANSACTION)
+                .entry( createEntry(measureReport) );
+        
+        if( resources != null ) {
+            resources.stream()
+                .map( p -> p.getResource() )
+                .forEach( r -> builder.entry( createEntry(r) ) );
+        }
+        
+        try {
+            Bundle response = resourceHelper.doBundle(builder.build(), false);
+            
+            return  Parameters.builder()
+                    .parameter( Parameter.builder().name(fhirstring(PARAM_OUT_RETURN)).resource(response).build())
+                    .build();
```

Comment:
```suggestion
            return  Parameters.builder()
                    .parameter( Parameter.builder().name(fhirstring(PARAM_OUT_RETURN)).resource(response).build())
                    .build();
```
Please use the FHIROperationUtil helper method that facilitates the return.